web/blueprints/user/__init__.py

```diff
@@ -142,15 +145,119 @@ def static_datasheet(user_id):
 
 @bp.route('/json/search')
 def json_search():
-    query = request.args['query']
-    results = session.session.query(User.id, User.login, User.name).filter(or_(
-        func.lower(User.name).like(func.lower(u"%{0}%".format(query))),
-        func.lower(User.login).like(func.lower(u"%{0}%".format(query))),
-        cast(User.id, Text).like(u"{0}%".format(query))
-    )).all()
-    users = [{"id": user_id, "login": login, "name": name}
-             for user_id, login, name in results]
-    return jsonify(users=users)
+    user_id = request.args.get('id')
+    name = request.args.get('name')
+    login = request.args.get('login')
+    mac = request.args.get('mac')
+    ip_address = request.args.get('ip_address')
+    trafficgroup_id = request.args.get('trafficgroup_id')
+    propertygroup_id = request.args.get('propertygroup_id')
+    building_id = request.args.get('building_id')
+    query = request.args.get("query")
+    result = User.q
+
+    applied_filter = True
```

Comment:
It was initialized with `False` in the past, but I decided to also show all results when no filter is given. I thought that it may needs to be reactivated, but I think that it can be removed.